sphinx/source/docs/dev_guide/notes.rst

```diff
@@ -0,0 +1,250 @@
+
+.. _devguide_notes:
+
+Notes
+=====
+
+This section collects miscellaneous information that may be of use to
+anyone developing Bokeh. Help organizing and improving this material is
+welcome.
+
+Environment Variables
+---------------------
+
+There are several environment variables that can be useful for developers:
+
+* ``BOKEH_BROWSER`` --- What browser to use when opening plots
+    Valid values are any of the browser names understood by the python
+    standard library webbrowser_ module.
+
+* ``BOKEH_LOCAL_DOCS_CDN`` --- What version of BokehJS to use when building  sphinx docs locally.
```

Comment:
je... It seems I loaded the file yesterday and I forgot to reload it this morning. I swear I checked the file and I did not see it!!! Thanks!